web/sites/default/config/search_api.index.full_index.yml

```diff
@@ -1078,7 +1092,6 @@ processor_settings:
       - group_topic_name
       - portland_content_type_name_field
       - portland_group_title_field
-      - rendered_item
```

Comment:
I remember originally this PR removed the rendered item Solr field but you've added it back, I'm curious did it have bad results?

Regardless, if we are keeping rendered_item we should probably add it back to the HTML filter Ignore characters filter here.